src/app/archive/lib/processor.ml

```diff
@@ -4867,8 +4867,14 @@ let setup_server ~proof_cache_db ~(genesis_constants : Genesis_constants.t)
       Deferred.unit
   | Ok pool ->
       let%bind () =
-        add_genesis_accounts pool ~logger ~genesis_constants
-          ~constraint_constants ~runtime_config_opt ~chunks_length
+        if skip_genesis_loading then (
+          [%log info]
+            "Skipping genesis accounts loading (--skip-genesis-loading flag \
+             set)" ;
+          Deferred.unit )
+        else
+          add_genesis_accounts pool ~logger ~genesis_constants
```

Comment:
i think when runtime_config_opt is null then no accounts will be added for genesis